Cargo.toml

```diff
@@ -1,18 +1,19 @@
 [package]
-name = "jobserver"
-version = "0.1.24"
-authors = ["Alex Crichton <alex@alexcrichton.com>"]
+name = "jobslot"
+version = "0.1.0"
+authors = ["Jiahao XU <Jiahao_XU@outlook.com>"]
```

Comment:
Unless you've rewritten absolutely all of it I'd say it's courteous to leave Alex in there, maybe as second author.

Reply:
Ok, I will add him back.